### PR TITLE
Add support for build-args

### PR DESCRIPTION
**Pull request by jaxxstorm**
It's common to want to add build args at build time. This adds support!

### CODE REVIEW

**Comment by jen20** [comment]
CI here is failing because of secrets not being available in a fork.

File: lib/main.js [hunk 1]
```diff
@@ -64,6 +64,10 @@ function readAndValidateConfig() {
             .split(",")
             .map(x => x.trim())
             .filter(x => !isNullOrWhitespace(x)),
+        buildArgs: core.getInput("build-args")
+            .split(",")
+            .map(x => x.trim())
+            .filter(x => isNullOrWhitespace(x)),
```
Comment (jen20) [inline lib/main.js:70]: This predicate seems like it might be the wrong way around?

File: lib/main.js [hunk 2]
```diff
@@ -134,6 +138,11 @@ function run() {
             for (const tag of effectiveAdditionalTags) {
                 buildParams.push("-t", `${config.repository}:${tag}`);
             }
+            if (!config.buildArgs.length) {
```
Comment (jen20) [inline lib/main.js:141]: I'd use a `for (const arg of config.buildArgs)` here without testing for length like this, I think?

**Review by jen20 — CHANGES_REQUESTED** [review]
Doh, just realised I reviewed the compiled form instead of the source form, but the comments are basically the same I think.

**Comment by jaxxstorm** [comment]
Closing this, will push to the remote

**Pull request closed without merging**